project/CrossJava.scala

```diff
@@ -31,16 +32,22 @@ case class JavaVersion(numbers: Vector[Long], vendor: Option[String]) {
 object JavaVersion {
 
   val specificationVersion: String = sys.props("java.specification.version")
+
   val version: String = sys.props("java.version")
-  val is1x: Boolean = version.startsWith("1.")
+
+  def isJdk8: Boolean =
+    VersionNumber(specificationVersion).matchesSemVer(SemanticSelector(s"=1.8"))
```

Comment:
stray string template `s`

Reply:
@johanandren Ouch, I can not believe I missed that, it's from a `${previousThing}`.